.changeset/fair-knives-relax.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': patch
```

Comment:
one more thing on this that I've forgotten.

I think this wants to be `'core-app-api': patch` only as we only change that package now 👍